js/packages/cli/src/helpers/upload/ipfs.ts

```diff
@@ -40,10 +41,35 @@ export async function ipfsUpload(
 
   await sleep(500);
 
+  let animationUrl = undefined;
```

Comment:
Most of this is copy paste from the `imageHash` work above. Can you refactor this?

Reply:
Sure, refactored aws as well